### ISSUE TITLE
Name property for Edit box in 'Package manager' window is not defined

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#362]
### Steps to Reproduce

1. Open "Xamarin Workbook"
2. Products window is opened.
3. Navigate to the "wpf "icon and select "create workbook".
4.Under "File" menu select "Add packages" or select 'Add package' button at the bottom of the workbook page.
5. Navigate to Edit box in 'Package manager' window and Observe Inspect property
### Expected Behavior
Name property should be properly defined for Edit box(For e.g -'Search Nugget' )
### Actual Behavior
Name property for Edit box is not defined and currently is " "

Narrator Behavior: Narrator is announcing as 'Editing, Newton, 0% vertically scrolled'
NVDA behavior:NVDA is announcing as 'edit, newton'

### User Impact:
Users who rely on screenreader will be unable to know the Name of Edit box

### Screen-Shots: 
![edit_name_property_package_window_inspect](https://user-images.githubusercontent.com/22956130/40724942-dd87d550-643f-11e8-940b-910361b7a9a2.png)
![edit_name_property_package_window_narrator](https://user-images.githubusercontent.com/22956130/40724943-ddb36684-643f-11e8-9e0d-b535b1969dda.png)
![edit_name_property_package_window_nvda](https://user-images.githubusercontent.com/22956130/40724945-dde39dd6-643f-11e8-970d-8a4dbe05882a.png)

### Environment: 
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1
